build qdrawhelper_avx2.cpp with -mavx2 rather than -march=core-avx2 to avoid compilation failures on non-AVX2 systems
cfr. https://bugreports.qt.io/browse/QTBUG-71564 (and https://gcc.gnu.org/bugzilla/show_bug.cgi?id=69471)
author: Kenneth Hoste (HPC-UGent)
--- qt-everywhere-src-5.12.3/qtbase/src/gui/Makefile.orig	2019-05-01 16:37:30.802829447 +0200
+++ qt-everywhere-src-5.12.3/qtbase/src/gui/Makefile	2019-05-01 16:38:30.204257192 +0200
@@ -3076,7 +3076,7 @@
 		painting/qdrawingprimitive_sse2_p.h \
 		painting/qrgba64_p.h \
 		.pch/Qt5Gui.gch/c++
-	g++ -c $(CXXFLAGS) -march=core-avx2 $(INCPATH) painting/qdrawhelper_avx2.cpp -o .obj/qdrawhelper_avx2.o
+	g++ -c $(CXXFLAGS) -mavx2 $(INCPATH) painting/qdrawhelper_avx2.cpp -o .obj/qdrawhelper_avx2.o
 
 compiler_arch_haswell_c_compiler_make_all:
 compiler_arch_haswell_c_compiler_clean:
